doc/snapper.xml.in

```diff
@@ -222,6 +222,30 @@
 	  <para>Specifies table style. Table style is identified by an integer number.</para>
 	</listitem>
       </varlistentry>
+      <varlistentry>
+	<term><option>--machine-readable</option></term>
+	<listitem>
+	  <para>Specifies a machine-readable output format. Possible options are csv and json.</para>
+	</listitem>
+      </varlistentry>
+      <varlistentry>
+	<term><option>--csvout</option></term>
+	<listitem>
+	  <para>Sets CSV output format.</para>
```

Comment:
As I mentioned earlier, CSV is deceptively simple but has multiple incompatible dialects (like regular expressions), therefore we should document the one we use:

```suggestion
	  <para>Sets CSV output format. See RFC 4180 for the details, except lines end with a LF, not a CR+LF.</para>
```

Let me look up the syntax for putting the https://tools.ietf.org/html/rfc4180 link in there. Or even better, an inline example:

```csv
record1,,,,,
record2,field with spaces,"field,with a comma","one double quote "" here","newline
here",field
record3,,,,,
```

Reply:
@joseivanlopez See https://github.com/openSUSE/snapper/commit/650a5a68d7b1163042b86ca0a4db862a903ddb90 (I could not add it to this PR because it is in your fork)